.storybook/stories/DataFiltering/components/DashboardPagination.tsx

```diff
@@ -38,25 +107,133 @@ export const DashboardPagination = () => {
 
 	if (!data.length) return null;
 
+	const hasChangedColumnSettings = !Object.values(
+		formState.activeColumns
+	).every(Boolean);
+	const hasChangedItemsPerPageSettings = pagination.perPage !== 10;
+	const appliedSettingsCount =
+		Number(hasChangedColumnSettings) + Number(hasChangedItemsPerPageSettings);
+
 	return (
-		<PaginationButtons
-			currentPage={displayText.page}
-			onChange={(page) => setPagination({ perPage: displayText.perPage, page })}
-			totalPages={totalPages}
-			itemRangeText={displayText.itemRangeText}
-			itemsPerPage={displayText.perPage}
-			onItemsPerPageChange={(perPage) =>
-				setPagination({
-					page: getValidPage({
-						currentPage: pagination.page,
-						totalItems,
-						perPage,
-						prevPerPage: pagination.perPage,
-					}),
-					perPage,
-				})
-			}
-		/>
+		<>
+			<Box
+				display="grid"
+				gap={1}
+				css={{
+					gridTemplateColumns: '1fr',
+					[tokens.mediaQuery.min.lg]: {
+						gridTemplateColumns: '2fr 1fr',
+					},
+				}}
+			>
+				<PaginationButtons
+					currentPage={displayText.page}
+					onChange={(page) =>
+						setPagination({ perPage: displayText.perPage, page })
+					}
+					totalPages={totalPages}
+					itemRangeText={displayText.itemRangeText}
+				/>
+
+				<Button
+					css={{
+						[tokens.mediaQuery.min.lg]: {
+							justifySelf: 'end',
+						},
+					}}
+					iconBefore={SettingsIcon}
+					onClick={openDrawer}
+					variant="tertiary"
+				>
+					Table settings ({appliedSettingsCount})
```

Comment:
Do you think we should have some visually hidden text describing what the `appliedSettingsCount` means? I don't know what that would be exactly though

Reply:
So that it reads out "Table settings, 2 applied settings"?

i.e. hide the bracketed visual text and add everything including and from the comma?

Reply:
yep, exactly - and great copy choice

Reply:
Just added this

Reply:
Thanks!